samples/metrics_monitor/CMakeLists.txt

```diff
@@ -35,11 +35,40 @@ if( PKG_LIBDRM_FOUND AND
     PKG_PCIACCESS_FOUND AND
     BUILD_TESTS)
 
+    # igtload library
+
+    include_directories (
+        ${PKG_LIBDRM_INCLUDE_DIRS}
+        ${CMAKE_CURRENT_SOURCE_DIR}/igtload/
+    )
+
+    link_libraries(
+        ${PKG_LIBDRM_LIBRARIES}
+        pthread 
+        rt
+    )
+
+    set( sources "" )
+
+    file( GLOB_RECURSE srcs "${CMAKE_CURRENT_SOURCE_DIR}/igtload/*.c" )
+    list( APPEND sources ${srcs})
+    
+    make_library( igtload none shared )
+    make_library( igtload_static none static )
```

Comment:
no. that's wrong thing to do. We don't need shared library for sure.

Reply:
fixed